EnglishTranslation-master/QUEST_LV_0200.tsv

```diff
@@ -1119,7 +1119,7 @@ QUEST_LV_0200_20150317_001118	Luckily, they did minimal damage. {nl}Now, if we c
 QUEST_LV_0200_20150317_001119	Restore the weakened seal towers using the symbol of Goddess Austeja.{nl}There is the Rankis Seal and the Ranka Seal.
 QUEST_LV_0200_20150317_001120	The demons found out about the source of evil energy and so relentlessly attacked to destroy the seal. {nl}I'm sure if the seal is strengthened well this time, they will retreat after a while.
 QUEST_LV_0200_20150317_001121	But we don't know what will happen next.{nl} We just have to believe that Goddess Austeja will take care of everything if anything happens.
```

Comment:
i know you didn't touch this line, but let's remove `But` at the start of this line as well.
maybe change it to `We don't know what will happen next, though.`